src/test/scala/latis/server/CapabilitiesServiceSpec.scala

```diff
@@ -0,0 +1,37 @@
+package latis.server
+
+import cats.effect.IO
+import io.circe._
+import io.circe.syntax._
+import org.http4s._
+import org.http4s.implicits._
+import org.scalatest.FlatSpec
+
+class CapabilitiesServiceSpec extends FlatSpec {
+
+  "The capabilities service" should "advertise all output options" in {
+    val req = Request[IO](Method.GET, Uri.uri("/capabilities"))
+
+    val expected = Json.obj(
+      ("HAPI", Json.fromString("2.0")),
+      ("status", Status.`1200`.asJson),
+      ("outputFormats", Json.fromValues(
+        List(
+          Json.fromString("binary"),
+          Json.fromString("csv"),
+          Json.fromString("json")
+        )
+      ))
+    )
+
+    val capabilities = new CapabilitiesService[IO]()
+
+    // The stream ought to contain only the body.
+    val (body :: Nil) =
```

Comment:
This is normally a bad thing to do because this will fail if the list is empty or has more than one thing in it. I'm surprised there's not a warning about it.

But, because this is a test, I think it's fine.

Reply:
I wouldn't worry about totality of code in tests. If the test fails due to an assumed result (e.g. bad pattern match) then it still fails. We may need to relax some compiler warnings on test code.